devices/JointTorqueControlDevice/src/JointTorqueControlDevice.cpp

```diff
@@ -50,6 +50,15 @@ template <class T> int findAndReturnIndex(std::vector<T>& v, T& x)
 }
 
 // INLINE OPERATIONS
+ /** Saturate the specified value between the specified bounds. */
+inline double saturation(const double x, const double jointVel, const double xMax, const double xMin)
+{
+if (jointVel > 10.0 || jointVel < -10.0)
+    return x > xMax ? xMax : (x < xMin ? xMin : x);
+else
+    return x;
```

Comment:
I would provide also the bounds of jointVel

Reply:
@GiulioRomualdi done in the latest commit.